### PR TITLE
Test api statistic

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: tests/Feature/StatisticApiTest.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    /**
+     * A basic test status code for statistics api.
+     *
+     * @return void
+     */
+    public function testStatusCode()
+    {
+        $response = $this->json('GET', '/api/statistics/counts');
+        $response->assertStatus(200);
+        $response = $this->json('GET', '/api/statistics/trends');
```
Comment (ghost) [inline tests/Feature/StatisticApiTest.php]: Don't check 2 cases in a function

File: tests/Feature/StatisticApiTest.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                'daily-menus',
+                'foods',
+                'materials',
+                'orders',
+                'suppliers',
+                'users'
+            ],
+            'success'
+        ]);
+        //test for trends statistic
```
Comment (ghost) [inline tests/Feature/StatisticApiTest.php]: Don't check 2 cases in a function.

File: tests/Feature/StatisticApiTest.php [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            'foods' => \App\Food::count(),
+            'materials' => \App\Material::count(),
+            'orders' => \App\Order::count(),
+            'suppliers' => \App\Supplier::count(),
+            'users' => \App\User::count()
+        ];
+        $arrayCompare = (object)$arrayCompare;
+        $this->assertEquals($data, $arrayCompare);
+
+        //test for trends api
```
Comment (ghost) [inline tests/Feature/StatisticApiTest.php]: Don't check 2 cases in a function.

**Review by ghost — COMMENTED** [review]
Please check my comment

**Comment by ghost** [comment]
Please add new doc to `resources/views/api-doc-builders/index.blade.php`

**Comment by hungbkdn** [comment]
I have updated, please check it again!
Thank you so much :smile:

**Comment by ghost** [comment]
LGTM :octocat:

**Pull request merged**